### ISSUE TITLE
✅ 🐞 Bug Report: Loop carousel with no value for "Number of slides" throws Fatal Error in 3.16

### ISSUE THREAD

**Issue by Femobility** [elementor/elementor#23804]
### Prerequisites

- [X] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [X] The issue still exists against the latest stable version of Elementor.

### Description

I had left my Loop carousel setting for "Number of slides" empty, since I want the carousel to display all blog posts. That caused a critical error on the site (I was notified by Wordpress via email).

This happened yesterday after updating Elementor Pro to 3.16, I rolled back to 3.15.1 and the site was working again. 
Today Pro 3.16.1 was released, I updated and got the same problem. When I gave the field a value (after rolling back again to be able to that) it is working fine again. 

### Steps to reproduce

1) Create a Loop carousel without giving it a value for "Number of slides", with Elementor Pro version 3.16 or 3.16.1

### Isolating the problem

- [X] This bug happens with only Elementor plugin active (and Elementor Pro).
- [X] This bug happens with a Blank WordPress theme active ([Hello theme](https://wordpress.org/themes/hello-elementor/)).
- [X] I can reproduce this bug consistently following the steps above.

### System Info

[system-info-www.femobility.se-14-09-2023.txt](https://github.com/elementor/elementor/files/12612223/system-info-www.femobility.se-14-09-2023.txt)

**Comment by toineenzo** [comment]
You could also try entering "-1" to show every post. Sometimes the input box cleared by default, so an empty input box shouldn't cause a critical error :S

**Comment by nicholaszein** [comment]
Hello, @Femobility! 

Thank you for submitting your issue, and apologies for the delay! 🙏 

Please allow us some time to investigate it, and get back to you with our findings. 🧐 

Best regards

**Comment by toineenzo** [comment]
@nicholaszein I have also encountered this bug. It only happens with elementor pro 3.16+ (core/free on 3.16+ doesn't matter). It happens when you have a loop carrousel that has no set the number of slides (because I wanted to show ALL slides). This will cause a critical error only on the page with this certain carroussel. See my Logs below



<details>
  <summary>Error</summary>
  ```
 Fatal error: Uncaught Error: count(): Argument #1 ($value) must be of type Countable|array, null given
in /home/u124288p120450/domains/pypsevents.nl/public_html/wp-content/plugins/elementor-pro/base/base-carousel-trait.php on line 1385

Call stack:

count()
wp-content/plugins/elementor-pro/base/base-carousel-trait.php:1385
ElementorPro\M\L\W\Loop_Carousel::get_num_of_carousel_items()
wp-content/plugins/elementor-pro/base/base-carousel-trait.php:1080
ElementorPro\M\L\W\Loop_Carousel::render_carousel_footer()
wp-content/plugins/elementor-pro/modules/loop-builder/widgets/loop-carousel.php:140
ElementorPro\M\L\W\Loop_Carousel::render_loop_footer()
wp-content/plugins/elementor-pro/modules/loop-builder/skins/skin-loop-base.php:193
ElementorPro\M\L\S\Skin_Loop_Base::render_loop_footer()
wp-content/plugins/elementor-pro/modules/posts/skins/skin-base.php:882
ElementorPro\M\P\S\Skin_Base::render()
wp-content/plugins/elementor-pro/modules/loop-builder/skins/skin-loop-base.php:89
ElementorPro\M\L\S\Skin_Loop_Base::render()
wp-content/plugins/elementor/includes/base/skin-base.php:79
Elementor\Skin_Base::render_by_mode()
wp-content/plugins/elementor/includes/base/widget-base.php:607
Elementor\Widget_Base::render_content()
wp-content/plugins/elementor/includes/base/widget-base.php:750
Elementor\Widget_Base::print_content()
wp-content/plugins/elementor/includes/base/element-base.php:464
Elementor\Element_Base::print_element()
wp-content/plugins/elementor/includes/base/element-base.php:1375
Elementor\Element_Base::print_content()
wp-content/plugins/elementor/includes/base/element-base.php:464
Elementor\Element_Base::print_element()
wp-content/plugins/elementor/core/base/document.php:1739
Elementor\C\B\Document::print_elements()
wp-content/plugins/elementor/core/base/document.php:1134
Elementor\C\B\Document::print_elements_with_wrapper()
wp-content/plugins/elementor/includes/frontend.php:1179
Elementor\Frontend::get_builder_content()
wp-content/plugins/elementor-pro/modules/posts/skins/skin-content-base.php:265
ElementorPro\M\T\W\Post_Content::render_post_content()
wp-content/plugins/elementor-pro/modules/theme-builder/widgets/post-content.php:111
ElementorPro\M\T\W\Post_Content::render()
wp-content/plugins/elementor/includes/base/controls-stack.php:2265
Elementor\Controls_Stack::render_by_mode()
wp-content/plugins/elementor/includes/base/widget-base.php:609
Elementor\Widget_Base::render_content()
wp-content/plugins/elementor/includes/base/widget-base.php:750
Elementor\Widget_Base::print_content()
wp-content/plugins/elementor/includes/base/element-base.php:464
Elementor\Element_Base::print_element()
wp-content/plugins/elementor/includes/base/element-base.php:1375
Elementor\Element_Base::print_content()
wp-content/plugins/elementor/includes/base/element-base.php:464
Elementor\Element_Base::print_element()
wp-content/plugins/elementor/core/base/document.php:1739
Elementor\C\B\Document::print_elements()
wp-content/plugins/elementor-pro/modules/theme-builder/documents/theme-document.php:418
ElementorPro\M\T\D\Theme_Document::print_elements_with_wrapper()
wp-content/plugins/elementor/includes/frontend.php:1179
Elementor\Frontend::get_builder_content()
wp-content/plugins/elementor/core/base/document.php:1183
Elementor\C\B\Document::get_content()
wp-content/plugins/elementor/modules/library/documents/library-document.php:76
Elementor\M\L\D\Library_Document::get_content()
wp-content/plugins/elementor-pro/modules/theme-builder/documents/theme-document.php:158
ElementorPro\M\T\D\Theme_Document::get_content()
wp-content/plugins/elementor-pro/modules/theme-builder/documents/theme-document.php:173
ElementorPro\M\T\D\Theme_Document::print_content()
wp-content/plugins/elementor-pro/modules/theme-builder/documents/single-base.php:93
ElementorPro\M\T\D\Single_Base::print_content()
wp-content/plugins/elementor-pro/modules/theme-builder/classes/locations-manager.php:364
ElementorPro\M\T\C\Locations_Manager::do_location()
wp-content/plugins/elementor-pro/modules/theme-builder/api.php:13
elementor_theme_do_location()
wp-content/themes/hello-elementor/index.php:20
include()
wp-includes/template-loader.php:106
require_once()
wp-blog-header.php:19
require()
index.php:17
  ```
</details>

<details>
  <summary>Elementor System Info</summary>
  ```

== Server Environment ==
	Operating System: Linux
	Software: Apache/2
	MySQL version: MariaDB Server v10.4.27
	PHP Version: 8.0.27
	PHP Memory Limit: 2048M
	PHP Max Input Vars: 3000
	PHP Max Post Size: 128M
	GD Installed: Yes
	ZIP Installed: Yes
	Write Permissions: All right
	Elementor Library: Connected

== WordPress Environment ==
	Version: 6.3.1
	Site URL: https://pypsevents.nl
	Home URL: https://pypsevents.nl
	WP Multisite: No
	Max Upload Size: 128 MB
	Memory limit: 768M
	Max Memory limit: 1024M
	Permalink Structure: /%postname%/
	Language: en-GB
	Timezone: Europe/Amsterdam
	Admin Email: bestuur@pypsevents.nl
	Debug Mode: Active

== Theme ==
	Name: Hello Elementor
	Version: 2.8.1
	Author: Elementor Team
	Child Theme: No

== User ==
	Role: administrator
	WP Profile lang: en_GB
	User Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/16.2 Safari/605.1.15

== Active Plugins ==
	Admin and Site Enhancements
		Version: 5.7.1
		Author: Bowo

	Admin Columns Pro
		Version: 6.3.3
		Author: AdminColumns.com

	Admin Menu Editor
		Version: 1.11.1
		Author: Janis Elsts

	Advanced Custom Fields: Extended
		Version: 0.8.9.3
		Author: ACF Extended

	Advanced Custom Fields PRO
		Version: 6.2.1.1
		Author: WP Engine

	Analytics Insights for Google Analytics 4 (AIWP)
		Version: 5.9.3
		Author: DeConf

	Antispam Bee
		Version: 2.11.4
		Author: pluginkollektiv

	BetterLinks
		Version: 1.5.17
		Author: WPDeveloper

	BetterLinks Pro
		Version: 1.6.1
		Author: WPDeveloper

	Complianz | GDPR/CCPA Cookie Consent
		Version: 6.5.3
		Author: Really Simple Plugins

	DynamicTags
		Version: 1.2.2
		Author: RTO GmbH

	Elementor
		Version: 3.16.3
		Author: Elementor.com

	Elementor Pro
		Version: 3.16.1
		Author: Elementor.com

	FluentSMTP
		Version: 2.2.5
		Author: FluentSMTP & WPManageNinja Team

	Health Check &amp; Troubleshooting
		Version: 1.7.0
		Author: The WordPress.org community

	Instant Images
		Version: 6.0.0
		Author: Darren Cooney

	iThemes Security
		Version: 8.1.8
		Author: iThemes

	Kadence WooCommerce Email Designer
		Version: 1.5.11
		Author: Kadence WP

	MainWP Child
		Version: 4.5.1
		Author: MainWP

	Manage Notification E-mails
		Version: 1.8.5
		Author: Virgial Berveling

	Meta pixel for WordPress
		Version: 3.0.13
		Author: Facebook

	OoohBoi Steroids for Elementor
		Version: 2.1.6
		Author: OoohBoi

	Query Monitor
		Version: 3.13.1
		Author: John Blackbourn

	Rank Math SEO
		Version: 1.0.201.1
		Author: Rank Math

	Really Simple SSL
		Version: 7.0.9
		Author: Really Simple Plugins

	Real Media Library (Free)
		Version: 4.21.9
		Author: devowl.io

	Redis Object Cache
		Version: 2.4.4
		Author: Till Krüss

	UiPress Lite
		Version: 3.2.12
		Author: Admin 2020

	WooCommerce
		Version: 8.1.1
		Author: Automattic

	WP Rocket
		Version: 3.14.4
		Author: WP Media

	WP Social Ninja
		Version: 3.10.1
		Author: WPManageNinja LLC

	WP Social Ninja Pro
		Version: 3.10.0
		Author: WPManageNinja LLC


== Must-Use Plugins ==
	Health Check Troubleshooting Mode
		Version: 1.8.1
		Author: 


== Features ==
	Custom Fonts: 3
	Custom Icons: 0

== Integrations ==
	
	google_maps: Active
	recaptcha: Active
	recaptcha_v3: Active
	facebook_sdk: Active
	woocommerce: Active


== Elementor Experiments ==
	Optimised DOM Output: Active by default
	Improved asset loading: Inactive
	Improved CSS Loading: Inactive
	Inline Font Icons: Active
	Additional Custom Breakpoints: Inactive
	admin_menu_rearrangement: Inactive by default
	Flexbox Container: Active
	Upgrade Swiper Library: Active
	Grid Container: Active
	Default to New Theme Builder: Active
	Hello Theme Header &amp; Footer: Inactive
	Editor Top Bar: Active
	Landing pages: Active by default
	Nested Elements: Active
	Lazy Load Background Images: Inactive by default
	Pages Panel: Inactive by default
	Global Style Guide: Active
	Page Transitions: Active by default
	Notes: Active
	Form Submissions: Active
	Scroll Snap: Active
	Menu: Active
	Taxonomy Filter: Active


== Log ==
	
Log: showing 20 of 342023-09-13 15:15:10 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor',
  'from' => '3.16.1',
  'to' => '3.16.2',
)]
2023-09-13 15:15:11 [info] elementor::elementor_updater Started 
2023-09-13 15:15:11 [info] Elementor/Upgrades - _on_each_version Start  
2023-09-13 15:15:11 [info] Elementor/Upgrades - _on_each_version Finished 
2023-09-13 15:15:11 [info] Elementor/Upgrades - _v_3_17_0_site_settings_updates Start  
2023-09-13 15:15:11 [info] Elementor/Upgrades - _v_3_17_0_site_settings_updates Finished 
2023-09-13 15:15:11 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.16.1',
  'to' => '3.16.2',
)]
2023-09-13 15:15:13 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor',
  'from' => '3.16.1',
  'to' => '3.16.2',
)]
2023-09-15 06:07:17 [info] Update DB has been started [array (
  'plugin' => 'Elementor',
  'from' => '3.16.2',
  'to' => '3.16.3',
)]
2023-09-15 06:07:17 [info] elementor::elementor_updater Started 
2023-09-15 06:07:17 [info] Elementor/Upgrades - _on_each_version Start  
2023-09-15 06:07:17 [info] Elementor/Upgrades - _on_each_version Finished 
2023-09-15 06:07:17 [info] Elementor/Upgrades - _v_3_17_0_site_settings_updates Start  
2023-09-15 06:07:17 [info] Elementor/Upgrades - _v_3_17_0_site_settings_updates Finished 
2023-09-15 06:07:17 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.16.2',
  'to' => '3.16.3',
)]
2023-09-15 06:07:41 [info] Update DB has been started [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.16.0',
  'to' => '3.16.1',
)]
2023-09-15 06:07:41 [info] elementor-pro::elementor_pro_updater Started 
2023-09-15 06:07:41 [info] Elementor Pro/Upgrades - _on_each_version Start  
2023-09-15 06:07:41 [info] Elementor Pro/Upgrades - _on_each_version Finished 
2023-09-15 06:07:41 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.16.0',
  'to' => '3.16.1',
)]

PHP: showing 6 of 6PHP: 2023-09-13 04:00:12 [warning X 46][/home/u124288p120450/domains/pypsevents.nl/public_html/wp-content/plugins/elementor-pro/modules/posts/skins/skin-content-base.php::265] Attempt to read property &quot;ID&quot; on null [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-09-13 04:28:11 [error X 3145][/home/u124288p120450/domains/pypsevents.nl/public_html/wp-content/plugins/elementor-pro/base/base-carousel-trait.php::1385] Uncaught TypeError: count(): Argument #1 ($value) must be of type Countable|array, null given in /home/u124288p120450/domains/pypsevents.nl/public_html/wp-content/plugins/elementor-pro/base/base-carousel-trait.php:1385
Stack trace:
#0 /home/u124288p120450/domains/pypsevents.nl/public_html/wp-content/plugins/elementor-pro/base/base-carousel-trait.php(1385): count()
#1 /home/u124288p120450/domains/pypsevents.nl/public_html/wp-content/plugins/elementor-pro/base/base-carousel-trait.php(1080): ElementorPro\Modules\LoopBuilder\Widgets\Loop_Carousel-&gt;get_num_of_carousel_items()
#2 /home/u124288p120450/domains/pypsevents.nl/public_html/wp-content/plugins/elementor-pro/modules/loop-builder/widgets/loop-carousel.php(140): ElementorPro\Modules\LoopBuilder\Widgets\Loop_Carousel-&gt;render_carousel_footer()
#3 /home/u124288p120450/domains/pypsevents.nl/public_html/wp-content/plugins/elementor-pro/modules/loop-builder/skins/skin-loop-base.php(193): ElementorPro\Modules\LoopBuilder\Widgets\Loop_Carousel-&gt;render_loop_footer()
#4 /home/u124288p120450/domains/pypsevents.nl/public_html/wp-content/plugins/elementor-pro/modules/posts/skins/skin-base.php(882): ElementorPro\Modules\LoopBuilder\Skins\Skin_Loop_Base-&gt;render_loop_footer()
#5 /home/u124288p120450/domains/pypsevents.nl/public_html/wp-content/plugins/elementor-pro/modules/loop-builder/skins/skin-loop-base.php(89): ElementorPro\Modules\Posts\Skins\Skin_Base-&gt;render()
#6 /home/u124288p120450/domains/pypsevents.nl/public_html/wp-content/plugins/elementor/includes/base/skin-base.php(79): ElementorPro\Modules\LoopBuilder\Skins\Skin_Loop_Base-&gt;render()
#7 /home/u124288p120450/domains/pypsevents.nl/public_html/wp-content/plugins/elementor/includes/base/widget-base.php(607): Elementor\Skin_Base-&gt;render_by_mode()
#8 /home/u124288p120450/domains/pypsevents.nl/public_html/wp-content/plugins/elementor/includes/base/widget-base.php(750): Elementor\Widget_Base-&gt;render_content()
#9 /home/u124288p120450/domains/pypsevents.nl/public_html/wp-content/plugins/elementor/includes/base/element-base.php(464): Elementor\Widget_Base-&gt;print_content()
#10 /home/u124288p120450/domains/pypsevents.nl/public_html/wp-content/plugins/elementor/includes/base/element-base.php(1375): Elementor\Element_Base-&gt;print_element()
#11 /home/u124288p120450/domains/pypsevents.nl/public_html/wp-content/plugins/elementor/includes/base/element-base.php(464): Elementor\Element_Base-&gt;print_content()
#12 /home/u124288p120450/domains/pypsevents.nl/public_html/wp-content/plugins/elementor/core/base/document.php(1739): Elementor\Element_Base-&gt;print_element()
#13 /home/u124288p120450/domains/pypsevents.nl/public_html/wp-content/plugins/elementor/core/base/document.php(1134): Elementor\Core\Base\Document-&gt;print_elements()
#14 /home/u124288p120450/domains/pypsevents.nl/public_html/wp-content/plugins/elementor/includes/frontend.php(1179): Elementor\Core\Base\Document-&gt;print_elements_with_wrapper()
#15 /home/u124288p120450/domains/pypsevents.nl/public_html/wp-content/plugins/elementor-pro/modules/posts/skins/skin-content-base.php(265): Elementor\Frontend-&gt;get_builder_content()
#16 /home/u124288p120450/domains/pypsevents.nl/public_html/wp-content/plugins/elementor-pro/modules/theme-builder/widgets/post-content.php(111): ElementorPro\Modules\ThemeBuilder\Widgets\Post_Content-&gt;render_post_content()
#17 /home/u124288p120450/domains/pypsevents.nl/public_html/wp-content/plugins/elementor/includes/base/controls-stack.php(2265): ElementorPro\Modules\ThemeBuilder\Widgets\Post_Content-&gt;render()
#18 /home/u124288p120450/domains/pypsevents.nl/public_html/wp-content/plugins/elementor/includes/base/widget-base.php(609): Elementor\Controls_Stack-&gt;render_by_mode()
#19 /home/u124288p120450/domains/pypsevents.nl/public_html/wp-content/plugins/elementor/includes/base/widget-base.php(750): Elementor\Widget_Base-&gt;render_content()
#20 /home/u124288p120450/domains/pypsevents.nl/public_html/wp-content/plugins/elementor/includes/base/element-base.php(464): Elementor\Widget_Base-&gt;print_content()
#21 /home/u124288p120450/domains/pypsevents.nl/public_html/wp-content/plugins/elementor/includes/base/element-base.php(1375): Elementor\Element_Base-&gt;print_element()
#22 /home/u124288p120450/domains/pypsevents.nl/public_html/wp-content/plugins/elementor/includes/base/element-base.php(464): Elementor\Element_Base-&gt;print_content()
#23 /home/u124288p120450/domains/pypsevents.nl/public_html/wp-content/plugins/elementor/core/base/document.php(1739): Elementor\Element_Base-&gt;print_element()
#24 /home/u124288p120450/domains/pypsevents.nl/public_html/wp-content/plugins/elementor-pro/modules/theme-builder/documents/theme-document.php(418): Elementor\Core\Base\Document-&gt;print_elements()
#25 /home/u124288p120450/domains/pypsevents.nl/public_html/wp-content/plugins/elementor/includes/frontend.php(1179): ElementorPro\Modules\ThemeBuilder\Documents\Theme_Document-&gt;print_elements_with_wrapper()
#26 /home/u124288p120450/domains/pypsevents.nl/public_html/wp-content/plugins/elementor/core/base/document.php(1183): Elementor\Frontend-&gt;get_builder_content()
#27 /home/u124288p120450/domains/pypsevents.nl/public_html/wp-content/plugins/elementor/modules/library/documents/library-document.php(76): Elementor\Core\Base\Document-&gt;get_content()
#28 /home/u124288p120450/domains/pypsevents.nl/public_html/wp-content/plugins/elementor-pro/modules/theme-builder/documents/theme-document.php(158): Elementor\Modules\Library\Documents\Library_Document-&gt;get_content()
#29 /home/u124288p120450/domains/pypsevents.nl/public_html/wp-content/plugins/elementor-pro/modules/theme-builder/documents/theme-document.php(173): ElementorPro\Modules\ThemeBuilder\Documents\Theme_Document-&gt;get_content()
#30 /home/u124288p120450/domains/pypsevents.nl/public_html/wp-content/plugins/elementor-pro/modules/theme-builder/documents/single-base.php(93): ElementorPro\Modules\ThemeBuilder\Documents\Theme_Document-&gt;print_content()
#31 /home/u124288p120450/domains/pypsevents.nl/public_html/wp-content/plugins/elementor-pro/modules/theme-builder/classes/locations-manager.php(364): ElementorPro\Modules\ThemeBuilder\Documents\Single_Base-&gt;print_content()
#32 /home/u124288p120450/domains/pypsevents.nl/public_html/wp-content/plugins/elementor-pro/modules/theme-builder/api.php(13): ElementorPro\Modules\ThemeBuilder\Classes\Locations_Manager-&gt;do_location()
#33 /home/u124288p120450/domains/pypsevents.nl/public_html/wp-content/themes/hello-elementor/index.php(20): elementor_theme_do_location()
#34 /home/u124288p120450/domains/pypsevents.nl/public_html/wp-includes/template-loader.php(106): include(&#039;/home/u124288p1...&#039;)
#35 /home/u124288p120450/domains/pypsevents.nl/public_html/wp-blog-header.php(19): require_once(&#039;/home/u124288p1...&#039;)
#36 /home/u124288p120450/domains/pypsevents.nl/public_html/index.php(17): require(&#039;/home/u124288p1...&#039;)
#37 {main}
  thrown [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-09-15 06:29:34 [notice X 1][/home/u124288p120450/domains/pypsevents.nl/public_html/wp-content/plugins/elementor/includes/base/controls-stack.php::228] hexdec(): Passing null to parameter #1 ($hex_string) of type string is deprecated [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-09-15 07:13:05 [notice X 1][/home/u124288p120450/domains/pypsevents.nl/public_html/wp-content/plugins/elementor-pro/modules/custom-css/module.php::76] trim(): Passing null to parameter #1 ($string) of type string is deprecated [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-09-19 02:23:24 [warning X 11][/home/u124288p120450/domains/pypsevents.nl/public_html/wp-content/plugins/elementor-pro/modules/dynamic-tags/acf/tags/acf-url.php::34] Undefined array key 0 [array (
  'trace' => '
#0: /home/u124288p120450/domains/pypsevents.nl/public_html/wp-content/plugins/elementor-pro/modules/dynamic-tags/acf/tags/acf-url.php(34): Elementor\Core\Logger\Manager -> rest_error_handler()
#1: /home/u124288p120450/domains/pypsevents.nl/public_html/wp-content/plugins/elementor/core/dynamic-tags/data-tag.php(44): ElementorPro\Modules\DynamicTags\ACF\Tags\ACF_URL -> get_value()
#2: /home/u124288p120450/domains/pypsevents.nl/public_html/wp-content/plugins/elementor/core/dynamic-tags/manager.php(216): Elementor\Core\DynamicTags\Data_Tag -> get_content()
#3: Elementor\Core\DynamicTags\Manager -> get_tag_data_content()
#4: /home/u124288p120450/domains/pypsevents.nl/public_html/wp-content/plugins/elementor/core/dynamic-tags/manager.php(109): class type call_user_func_array()
',
)]
PHP: 2023-09-19 02:23:24 [warning X 11][/home/u124288p120450/domains/pypsevents.nl/public_html/wp-content/plugins/elementor-pro/modules/dynamic-tags/acf/tags/acf-url.php::34] Undefined array key 1 [array (
  'trace' => '
#0: /home/u124288p120450/domains/pypsevents.nl/public_html/wp-content/plugins/elementor-pro/modules/dynamic-tags/acf/tags/acf-url.php(34): Elementor\Core\Logger\Manager -> rest_error_handler()
#1: /home/u124288p120450/domains/pypsevents.nl/public_html/wp-content/plugins/elementor/core/dynamic-tags/data-tag.php(44): ElementorPro\Modules\DynamicTags\ACF\Tags\ACF_URL -> get_value()
#2: /home/u124288p120450/domains/pypsevents.nl/public_html/wp-content/plugins/elementor/core/dynamic-tags/manager.php(216): Elementor\Core\DynamicTags\Data_Tag -> get_content()
#3: Elementor\Core\DynamicTags\Manager -> get_tag_data_content()
#4: /home/u124288p120450/domains/pypsevents.nl/public_html/wp-content/plugins/elementor/core/dynamic-tags/manager.php(109): class type call_user_func_array()
',
)]

JS: showing 2 of 2JS: 2023-09-15 04:29:20 [error X 1][https://pypsevents.nl/wp-admin/load-scripts.php?c=0&amp;loadchunk_0=jquery-core,jquery-migrate,utils,wp-polyfill-inert,regenerator-runtime,wp-polyfill,wp-hooks&amp;ver=6.3.1:2:28729] undefined is not an object (evaluating &#039;l[0].value&#039;) 
JS: 2023-09-15 04:36:35 [error X 1][https://pypsevents.nl/wp-content/plugins/elementor/assets/js/editor.min.js?ver=3.16.3:3:804285] undefined is not an object (evaluating &#039;this.store.find((function(T){return T.original===C})).localized&#039;) 



== Elementor - Compatibility Tag ==
	
	Elementor Pro: Compatibility not specified
	OoohBoi Steroids for Elementor: Compatibility not specified

== Elementor Pro - Compatibility Tag ==
	
  ```
</details>

**Comment by MIBul** [comment]
It is breaking the editor as well so you can't update the number of slides unless roll back to previous version. 
Can we have an update on this?

**Comment by nicholaszein** [comment]
He everyone!

We have great news! 🙌 

### 📢 We're happy to announce that the issue you raised was resolved in Elementor Pro v3.16.2! 🥳 

✅ Feel free to check it out and update your plugin to the new version!

> _Check the changelog for more information:_
> [Changelog of Elementor and Elementor Pro](https://elementor.com/help/changelog-of-elementor/)

Cheers 🥂
